apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -35,272 +34,78 @@ class ProgressBubble extends React.Component {
       PropTypes.string,
       PropTypes.number
     ]),
-    // This prop is provided as a testing hook, in normal use it will just be
-    // set to window.location; see defaultProps.
-    currentLocation: PropTypes.object.isRequired,
-    lessonTrophyEnabled: PropTypes.bool,
-    pairingIconEnabled: PropTypes.bool,
     hideToolTips: PropTypes.bool,
-    hideAssessmentIcon: PropTypes.bool,
     onClick: PropTypes.func
   };
 
-  static defaultProps = {
-    currentLocation: window.location
-  };
-
-  recordProgressTabProgressBubbleClick = () => {
-    firehoseClient.putRecord(
-      {
-        study: 'teacher_dashboard_actions',
-        study_group: 'progress',
-        event: 'go_to_level',
-        data_json: JSON.stringify({
-          student_id: this.props.selectedStudentId,
-          level_url: this.props.level.url,
-          section_id: this.props.selectedSectionId
-        })
-      },
-      {includeUserId: true}
-    );
-  };
-
-  render() {
-    const {
-      level,
-      disabled,
-      smallBubble,
-      selectedSectionId,
-      selectedStudentId,
-      currentLocation,
-      lessonTrophyEnabled,
-      pairingIconEnabled,
-      hideAssessmentIcon,
-      onClick
-    } = this.props;
-
-    const levelIsAssessment = isLevelAssessment(level);
-
-    const number = level.levelNumber;
-    const url = level.url;
-    const levelName = level.name || level.progressionDisplayName;
-    const levelIcon = getIconForLevel(level);
-
-    const hideNumber =
-      level.letter || level.isLocked || level.paired || level.bonus;
-
-    const style = {
-      ...styles.main,
-      ...(!disabled && hoverStyle),
-      ...(smallBubble && styles.small),
-      ...(level.isConceptLevel &&
-        (smallBubble ? styles.smallDiamond : styles.largeDiamond)),
-      ...levelProgressStyle(level.status, level.kind),
-      ...(level.highlighted && styles.highlighted)
-    };
-
-    let href = '';
-    if (!disabled && url && !onClick) {
-      const queryParams = queryString.parse(currentLocation.search);
+  isEnabled() {
```

Comment:
A more descriptive name for this might be `isBubbleClickable` or `isClickable`

Reply:
👍